servicebox-app/src/test/java/io/halkyon/ClaimsEndpointTest.java

```diff
@@ -29,6 +35,32 @@ public void testAddClaim(){
                 .extract().body().jsonPath().getString("id");
     }
 
+    @Test
+    public void testQueryUsingNameToGetClaims(){
+        given().header("HX-Request","true")
+               .queryParam("name","mysql-demo")
+               .when()
+                 .get("/claims/filter")
+               .then()
+                 .body(containsString("<td>mysql-demo</td>"));
+
+         /*RequestSpecification httpRequest = RestAssured.given().header("HX-Request","true").queryParam("name","mysql-demo");
+         Response response = httpRequest.get("/claims/filter");
+         ResponseBody body = response.getBody();
+         System.out.println("Response Body is: " + body.asString());
+         */
```

Comment:
Please, delete the commented code.

Reply:
I was keeping it to avoid to have to copy/paste it when t is needed to debug a HTTP request. Do you prefer that we move the code to a new test method ?

Reply:
See: c40f5ed